pyramid_zipkin/tween.py

```diff
@@ -106,6 +107,11 @@ def zipkin_tween(handler, registry):
     def tween(request):
         zipkin_settings = _get_settings_from_request(request)
 
+        if hasattr(request, 'yelp_context'):
+            context_stack = request.yelp_context.zipkin_context
```

Comment:
This is too Yelp-specific. We could add a pyramid setting to make it configurable.

Reply:
This is correct. It should go in `_get_settings_from_request` the same way we inject a transport_handler.